frontend/src/container/Header/index.tsx

```diff
@@ -135,16 +138,17 @@ function HeaderContainer(): JSX.Element {
 		<>
 			{showTrialExpiryBanner && (
 				<div className="trial-expiry-banner">
-					You are in free trial period. Your free trial will end on
+					You are in free trial period. Your free trial will end on{' '}
 					<span>
 						{getFormattedDate(licenseData?.payload?.trialEnd || Date.now())}.
 					</span>
 					{role === 'ADMIN' ? (
 						<span>
-							Please
-							<Button className="upgrade-link" type="link" onClick={handleUpgrade}>
+							{' '}
+							Please{' '}
```

Comment:
text might have issues can we wrap under Typography

Reply:
There are computations and nested spans required in the header, hence handling the spacing where data is conditionally computed.